frontend/src/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import { List, ListItem, ListItemButton, ListItemText, Box, Drawer, Collapse } from '@mui/material'
+import { ExpandLess, ExpandMore } from '@mui/icons-material';
+import { UserType, UserTypeProps } from '../types/global.d'
+import { useTranslation } from 'react-i18next'
+import { TFunction } from 'i18next'
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useLocation } from 'react-router-dom';
+
+const MenuButton = (text: string, path: string, t : TFunction, padding=2) : JSX.Element => (
```

Comment:
I set default `padding=2` because it's normal padding for simple menu buttons, that isn't in DropDownList
You can check, that when padding=0, it looks awful